comments.php

```diff
@@ -10,7 +10,6 @@
  */
 
 if ( have_comments() ) :
-	if ( ( is_page() || is_single() ) && ( ! is_home() && ! is_front_page() ) ) :
 ?>
 	<section id="comments">
 		<?php
```

Comment:
Should we add an `<hr>` here? I think it makes the the comments section easier to decipher from the post content with one.

Reply:
Maybe a good idea but also we still could add a line with CSS and border, which will avoid adding hr element...

Reply:
We are already using an `<hr>` element between the comments and the leave a reply form. 

**Questions for everybody:**
1. Do we want a line in both places or just one of the places? (between post and comments, and between comments and comment reply form)
2. Should we make this line with CSS or with `<hr>`?

Reply:
I would either remove the <hr> in both places or just add the <hr> element. I lean towards being less opinionated when it comes to design.